metpy/calc/thermo.py

```diff
@@ -1205,3 +1205,43 @@ def _isen_iter(iter_log_p, isentlevs_nd, ka, a, b, pok):
 
     # output values as a list
     return ret
+
+
```

Comment:
Or do we go to the convention that all functions called `cape` return both CAPE and CIN....and rename `cape_cin` to `cape`?